### ISSUE TITLE
[BUG] You cannot cheat items in survival, even if cheats are on.

### ISSUE THREAD

**Issue by yum13241** [CleanroomMC/HadEnoughItems#70]
How to reproduce:

1. Launch the game with HEI installed.
2. Create a world with cheats on, but in survival.
3. Ctrl+Click the wrench to enable cheat mode.
4. Try to cheat items.
5. Instead of getting the items you wanted, you get an error in chat and **64** Nether Stars.

Expected behavior:
I get the items I asked for.

Actual behavior:
I get a game breaking amount of Nether Stars.

**Comment by Rongmario** [comment]
Cannot replicate. It works as intended.

**Comment by yum13241** [comment]
Huh, something's up with my modpack.

**Comment by Rongmario** [comment]
If you do get an error, post your log here, the log should have a more explicit error.

**Comment by yum13241** [comment]
OK, I'll get to it.

Log: https://privatebin.net/?5675fc773460d796#2RK1QoKsLw2cm3V5h9c2KMF6beP7j2z9bPTGnn1rmi6L

(the first time I tried to cheat stuff in survival, I got smooth sandstone, which is what I asked for. But if I try again, not only does Ctrl+Click-ing the wrench give me nether stars (it got disabled), enabling it through any other means does not work (I recall it working once, then I would still get nether stars when I click something).
